docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
+
+\*\*: *Although GitHub can be considered fairly reliable, we highly recommend that you implement a solution to encrypt your collection. This is, after all, a way to your innermost thoughts.*
 
 #### Installing & Setting up Git with GitHub
 
-1. Create a [GitHub account](https://github.com/join/)
-2. [Follow this guide](https://vladmihalcea.com/tutorials/git/windows-git-ssh-authentication-to-github/) to:
-  - Install Git
-  - Setup a SSH key that automatically authenticate your Git installation with your GitHub account
+!> If you want to use SSH authentication instead of HTTPS, follow [this guide](https://vladmihalcea.com/tutorials/git/windows-git-ssh-authentication-to-github/)
+
+1. Create a [GitHub account](https://github.com/join/).
+
+2. Download [Git for Windows](https://gitforwindows.org/).
+
+3. Install it with default settings
+
+5. Open a command prompt: type `cmd.exe`, then press <kbd>Enter</kbd> and run the following two commands (with your own name and email you use for github):
+
+   ```
+   git config --global user.name "Piotr Wozniak"
+   git config --global user.email woz@supermemo.org
+   ```
+   Setup is now complete!
   
 #### Creating & Synchronizing your GitHub repository
 
-?> **In this example, the GitHub repository name will be named SuperMemo-Collection**
+?> **In this example, the GitHub repository we create will be named SuperMemo-Collection**
 
-1. Create a [new repository](https://github.com/new)
+1. Create a [new repository](https://github.com/new):
   - Give a name to your new repo (e.g. *SuperMemo-Collection*)
   - Select **Private**
+  - It is recommended to click **Initialize this repository with a README** to be able to immediately clone to computer
+
+<img src="/content/images/backup-setup/gitsetup-initialize.png" data-origin="content/images/backup-setup/gitsetup-initialize.png" alt="">
+
 2. In your SuperMemo Drive (**E:\\**), open a command prompt: type `cmd.exe`, then press <kbd>Enter</kbd>
 
 <img src="/content/images/backup-setup/windows-explorer-cmd.png" data-origin="content/images/backup-setup/windows-explorer-cmd.png" alt="">
 
-3. Go to your **GitHub repository** web page, click the <kbd>**Clone or download**</kbd> button, and copy the link.
+3. Go to your **GitHub repository** web page, click the <kbd>**Clone or download**</kbd> button, and **press use HTTPS**.
 
 <img src="/content/images/backup-setup/github-clone-link.png" data-origin="content/images/backup-setup/github-clone-link.png" alt="">
 
-4. In the **command prompt**, type `git clone <git@github.com:......>`. Replace the text between **< >** with the link you copied from GitHub.
-5. Your repository is now synchronized with your computer. Copy your SuperMemo collection in the new folder. Your folder should look similar to the example below (*.gitignore* might be missing):
+4. Copy the link starting with https://
```

Comment:
Suggestion: Highlight the `https://` with back quotes. That's also consistent with the wiki style